packages/component/src/components/tour-modal/tour-modal.tsx

```diff
@@ -27,73 +35,73 @@ export const TourModal: FC<TourModalProps> = ({ open, onClose }) => {
+          <div className={containerStyle}>
+            <div className={arrowStyle} onClick={() => setStep(0)}>
+              <ArrowLeftSmallIcon />
             </div>
             <div className={videoContainerStyle}>
-              <video
-                autoPlay
-                muted
-                loop
-                className={videoStyle}
-                data-testid="onboarding-modal-editing-video"
-              >
-                <source src="/editingVideo.mp4" type="video/mp4" />
-                <source src="/editingVideo.webm" type="video/webm" />
-                Create structured documents with ease, using a modular interface
-                to drag and drop blocks of text, images, and other content.
-              </video>
+              <div className={videoSlideStyle}>
+                <video
+                  autoPlay
+                  muted
+                  loop
+                  className={clsx(videoStyle, {
+                    [videoActiveStyle]: step === 0,
+                  })}
+                  data-testid="onboarding-modal-editing-video"
+                >
+                  <source src="/editingVideo.mp4" type="video/mp4" />
+                  <source src="/editingVideo.webm" type="video/webm" />
+                </video>
+                <video
+                  autoPlay
+                  muted
+                  loop
+                  className={clsx(videoStyle, {
+                    [videoActiveStyle]: step === 1,
+                  })}
+                  data-testid="onboarding-modal-switch-video"
+                >
+                  <source src="/switchVideo.mp4" type="video/mp4" />
+                  <source src="/switchVideo.webm" type="video/webm" />
+                </video>
+              </div>
             </div>
-            <div className={buttonContainerStyle}>
-              <Button
-                className={buttonStyle}
-                onClick={handleClose}
-                data-testid="onboarding-modal-ok-button"
-              >
-                Okay, I Like It !
-              </Button>
+            <div className={arrowStyle} onClick={() => setStep(1)}>
+              <ArrowRightSmallIcon />
             </div>
           </div>
-        )}
+          <ul className={tabContainerStyle}>
+            <li
+              className={clsx(tabStyle, { [tabActiveStyle]: step === 0 })}
+              onClick={() => setStep(0)}
+            ></li>
+            <li
+              className={clsx(tabStyle, { [tabActiveStyle]: step === 1 })}
+              onClick={() => setStep(1)}
+            ></li>
+          </ul>
+          <div className={descriptionStyle}>
+            {step === 1
+              ? ' Create structured documents with ease, using a modular interface to drag and drop blocks of text, images, and other content.'
+              : 'Easily switch between Page mode for structured document creation and Whiteboard mode for the freeform visual expression of creative ideas.'}
```

Comment:
Maybe add these to i18n?

Reply:
Of course, I need to carefully consider what names to give them.